src/components/password/password.pw.tsx

```diff
@@ -0,0 +1,556 @@
+        mount,
+        page,
+      }) => {
+        await mount(<PasswordComponent size={size} />);
+
+        await expect(password(page)).toHaveCSS("min-height", height);
+
+        const tokenValues = await getDesignTokensByCssProperty(
+          page,
+          password(page),
+          "min-height"
+        );
+        expect(tokenValues[0]).toBe(token);
+      });
+    }
+  );
+
+  specialCharacters.forEach((specificValue) => {
+    test(`should render with label prop as ${specificValue}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PasswordComponent label={specificValue} />);
+
+      const label = getDataElementByValue(page, "label");
+      await expect(label).toHaveText(specificValue);
+    });
+  });
+
+  specialCharacters.forEach((specificValue) => {
+    test(`should render with fieldHelp prop as ${specificValue}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PasswordComponent fieldHelp={specificValue} />);
+      await expect(fieldHelpPreview(page)).toHaveText(specificValue);
+    });
+  });
+
+  specialCharacters.forEach((specificValue) => {
+    test(`should render with labelHelp prop as ${specificValue}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PasswordComponent labelHelp={specificValue} />);
+
+      await getDataElementByValue(page, "question").hover();
+      await expect(tooltipPreview(page)).toHaveText(specificValue);
+    });
+  });
+
+  test("should render with an input icon", async ({ mount, page }) => {
+    await mount(<PasswordComponent inputIcon="add" />);
+    await expect(getDataElementByValue(page, "add")).toBeVisible();
+  });
+
+  test("should render with required prop", async ({ mount, page }) => {
+    await mount(<PasswordComponent required />);
+    await verifyRequiredAsteriskForLabel(page);
+  });
+
+  test("should render with autofocus prop", async ({ mount, page }) => {
+    await mount(<PasswordComponent autoFocus />);
+    await expect(passwordInput(page)).toBeFocused();
+  });
+
+  ([
+    ["left", "start"],
+    ["right", "end"],
+  ] as [TextboxProps["labelAlign"], string][]).forEach(
+    ([labelAlign, cssValue]) => {
+      test(`should render with ${labelAlign} labelAlign prop`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<PasswordComponent labelInline labelAlign={labelAlign} />);
+
+        const labelParent = getDataElementByValue(page, "label").locator("..");
+        await expect(labelParent).toHaveCSS(
+          "justify-content",
+          `flex-${cssValue}`
+        );
+      });
+    }
+  );
+
+  ["10%", "30%", "50%", "80%", "100%"].forEach((maxWidth) => {
+    test(`should render with ${maxWidth} max-width`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PasswordComponent maxWidth={maxWidth} />);
+
+      const passwordParent = password(page).locator("..");
```

Comment:
Should this be:
`const passwordParent = password(page).locator("..").locator("..");`
Cypress test has:
`getDataElementByValue("input").parent().parent()`

Reply:
the input.parent() is password/texbox(page), so we can just use .locator(..) once here